lib/main.dart

```diff
@@ -60,7 +81,7 @@ class MyApp extends StatelessWidget {
                 if (snapshot.connectionState == ConnectionState.active) {
                   final bool isloggedin = snapshot.hasData;
                   if (isloggedin == true) {
-                    return Dashboard();
+                     return Dashboard(user: userinfo,);
```

Comment:
Use the snapshot.data.uid and pass it to the Dashboard and pass the uid to fetch the user data